### PR TITLE
Commit number 1

### PR DESCRIPTION
**Pull request by kompotkot**
Some description due opening PR 11

### CODE REVIEW

**Comment by kompotkot** [comment]
New published comment

**Comment by kompotkot** [comment]
Some conversation

**Comment by kompotkot** [comment]
Some conversation

**Comment by kompotkot** [comment]
Some conversation

**Comment by kompotkot** [comment]
Some conversation

**Comment by kompotkot** [comment]
I should add jira ticket

**Comment by kompotkot** [comment]
Another one

**Comment by kompotkot** [comment]
Add another

**Comment by kompotkot** [comment]
Add more information

**Comment by kompotkot** [comment]
More information

**Comment by kompotkot** [comment]
One

**Comment by kompotkot** [comment]
Another one

**Comment by kompotkot** [comment]
Add more

**Comment by kompotkot** [comment]
More questions

**Comment by kompotkot** [comment]
Added another

**Comment by kompotkot** [comment]
Run alembic

**Comment by kompotkot** [comment]
@kompotkot check
- [ ] Alembic migration

**Comment by kompotkot** [comment]
Leave a comment

**Comment by kompotkot** [comment]
Leave another comment

**Comment by kompotkot** [comment]
Add additional comment

**Comment by kompotkot** [comment]
Add additional comment

**Comment by kompotkot** [comment]
One more

**Comment by kompotkot** [comment]
Another one

**Comment by kompotkot** [comment]
Add comment

**Comment by kompotkot** [comment]
New one

**Comment by kompotkot** [comment]
AWS

**Comment by kompotkot** [comment]
More

**Comment by kompotkot** [comment]
Hell

**Comment by kompotkot** [comment]
Add comment

**Comment by kompotkot** [comment]
New one

**Comment by kompotkot** [comment]
More ocmments

**Comment by kompotkot** [comment]
Another comment

**Comment by kompotkot** [comment]
More comments

**Comment by kompotkot** [comment]
New one

**Comment by kompotkot** [comment]
More text

**Comment by kompotkot** [comment]
Info info

**Comment by kompotkot** [comment]
Few comments created aaa

**Comment by kompotkot** [comment]
More and more 111

**Comment by kompotkot** [comment]
Hello again

File: test/2.py [hunk 1]
```diff
@@ -3,10 +3,14 @@
 @bugout-dev locust report
 Return me report @bugout-dev locust
 """
+class A:
+    """
+    Some description of class.
```
Comment (kompotkot) [inline test/2.py]: Inner

**Comment by kompotkot** [comment]
More comments 1

**Comment by kompotkot** [comment]
More comments
changed

**Comment by kompotkot** [comment]
Another

**Comment by kompotkot** [comment]
123

**Comment by kompotkot** [comment]
More comments

**Comment by kompotkot** [comment]
More comments 2 42

**Comment by serhgmail** [comment]
Looks good

**Comment by serhgmail** [comment]
@kompotkot check
- [ ] Run alembic migration
- [ ] Add AWS variables

**Comment by serhgmail** [comment]
@kompotkot check
- [ ] Run alembic migration
- [ ] Add AWS variables

**Comment by serhgmail** [comment]
@kompotkot check
- [ ] Run alembic migration
- [ ] Add AWS variables

**Comment by serhgmail** [comment]
@kompotkot check
- [x] set AWS variables

**Comment by kompotkot** [comment]
More text

**Comment by serhgmail** [comment]
Comments flow

**Comment by serhgmail** [comment]
@kompotkot check
- [ ] Run migration

**Comment by serhgmail** [comment]
Comments 1

**Comment by serhgmail** [comment]
Comments 23

**Comment by serhgmail** [comment]
New comm

**Comment by serhgmail** [comment]
More and more

**Comment by serhgmail** [comment]
More time changed

**Comment by serhgmail** [comment]
New one

**Comment by kompotkot** [comment]
123

**Comment by kompotkot** [comment]
More comments

**Comment by serhgmail** [comment]
More tests 123

**Comment by serhgmail** [comment]
hope

**Comment by serhgmail** [comment]
Additional information

**Comment by serhgmail** [comment]
@kompotkot check
- [x] Run migration
- [ ] Add vars

**Comment by kompotkot** [comment]
Valuable info

**Comment by serhgmail** [comment]
@kompotkot summarize

**Comment by serhgmail** [comment]
Check update old one

**Comment by serhgmail** [comment]
@kompotkot summarize

**Comment by kompotkot** [comment]
@kompotkot check
- [ ] Remove duplication

**Pull request closed without merging**